OpenRVS/classes/OpenServer.uc

```diff
@@ -130,13 +72,9 @@ function Tick(float delta)
 		//but lag for high-ping players always meant that the enemy would "pop" in
 		//this fix requires marginally more bandwidth but forces the server to replicate all alive pawns at all times
 		Pawn = C.Pawn;
-		if ( Pawn != none )
+		if ( Pawn != none ) && if ( Pawn.IsAlive() ) && ( !Pawn.bAlwaysRelevant )

```

Comment:
You can only have one "if" in a line, you'll need to delete the second.  So it looks like:
if ( Pawn != none ) && ( Pawn.IsAlive() ) && ( !Pawn.bAlwaysRelevant )

Reply:
thanks, that's a legit bug you caught!